docs/CHANGELOG.md

```diff
@@ -0,0 +1,147 @@
+## canary -> pre-canary
+- Remove `onCellCopyPaste` prop
+- Remove `getValidFilterValues` prop
+- Replace `onCellSelected` and `onCellDeSelected` with `onSelectedCellChange` prop
+- Replace `cellRangeSelection.onStart/onUpdate/onEnd` props with a single `onCellRangeSelectionChange` prop
+- Add `filters` and `onFiltersChange` props
+- Add `enableCellCopyPaste` prop to toggle copy/paste
+- Add `enableCellDragAndDrop` prop to toggle drag fill
+- Remove `fromRowId`, `toRowId`, and `fromRowData` from `onGridRowsUpdated` argument
+- Remove `column.getRowMetaData` prop
+- Header now only render visible cells [1837](https://github.com/adazzle/react-data-grid/pull/1837)
+- Replace `scrollToRowIndex` prop with `scrollToRow` method
+- Row and Cell renderers no longer receive `cellMetaData` prop
+- Remove `getCellActions` prop. Check [here](https://github.com/adazzle/react-data-grid/pull/1845) on how to migrate
+- Remove `react-data-grid-addons` package.
+- Remove subrow props (`getSubRowDetails`, `onCellExpand`, `onDeleteSubRow`, and `onAddSubRow`). Check [here](https://github.com/adazzle/react-data-grid/pull/1853) on how to migrate
+- Remove `cellContentRenderer`
+- `column.editor`, `rowRenderer`, and `headerRenderer` no longer support `ReactElement`
+- `column.formatter` no longer receives a `value` prop.
+- Remove `column.events` prop
+- Replace `rowGetter` and `rowsCount` props with a single `rows` prop
+- Rename `minHeight` to `height`
+- Rename `minWidth` to `width`
+- Rename `RowsContainer` to `rowsContainer`
+- Rename `onGridSort` to `onSort`
+- Rename `onGridRowsUpdated` to `onRowsUpdate`
+
+## next -> canary
+- Column resize now resizes all the cells which means re-rendering the whole grid when resizing columns.
+- Add `column.cellContentRenderer`. This is an opt-in feature, so it should not break anything. More info in [here](https://gist.github.com/nstepien/090298c3c2d94324cb332c33d82fdcfb)
+- `ReactDataGrid` component is renamed to `DataGrid`
```

Comment:
Do we really need to mention this? I mean this seems something internal as they can still import like 
```
import ReactDataGrid from 'react-data-grid';
```

Reply:
We don't need it as it is a default import. I just added it to emphasize that we should use `import DataGrid` instead of `import ReactDataGrid`. We can remove it from the final release notes